### DISCUSSION TITLE
Kandinsky 2.1 support

### DISCUSSION THREAD

**Discussion by adrianpuiu** [vladmandic/sdnext#263] [General]
Would id be possible to also support the new released Kandinsky 2.1 ? 

here are the safetensor files after conversion : 
https://huggingface.co/dreamlike-art/kandinsky-2.1/tree/main 

Here's the main git : 
https://github.com/ai-forever/Kandinsky-2

**Comment by ke1ne** [comment]
Kandinsky model is a product of the Russian goverment bank team. Supporting them is a supporting terrorist state

**Reply by myndxero** [reply]
Get out of here with that political crap.

**Reply by ke1ne** [reply]
I'm pretty sure that magic formula help you against armed soldiers who are raping you. Good luck

**Reply by myndxero** [reply]
You should probably do some research, current Russia is of the US's own making. When the USSR was desolved they were more than ready to join Westerners in capitalism. But Bush then Clinton and Obama all stone-walled them from joining Nato, despite the fact that it was the offering if they reduced their nuclear arsenal for which they complied.

Putin sucks and is an asshole, and his obsession with super long tables is weird, but it was a monster of our own making, just like weak US leadership was the opportunity Putin needed to invade Crimea, which btw, has ALWAYS belonged to Russia since Russia was a thing, until we took it away. Would be similar to telling the Jews, too bad, you can't have Israel anymore.

And btw, US further exacerbated the conflict over Ukraine suggested to join Nato, that has no right to be in Nato. They're not a democracy, one of the most corrupt countries on the planet where we send our money to get laundered.

So yeah, I don't know much about Kandinsky, I don't care, it's free and open source, so not only are you an idiot for suggesting using it SUPPORTS a 'terrorist state,'  you know absolutely nothing about the politics behind it. I may have missed some things, but lightyears ahead of your intelligent "magic formula help you against armed soldiers who are raping" cuz we already have that mindset infecting our population.

.fin

**Reply by ke1ne** [reply]
Well, you're quoting the russian state propaganda word by word: bad US, NATO expansion, blahblah. Nothing to discuss here. Russia is the civilized world enemy, and it's the main point here.

**Reply by myndxero** [reply]
> Well, you're quoting the russian state propaganda word by word: bad US, NATO expansion, blahblah. Nothing to discuss here. Russia is the civilized world enemy, and it's the main point here.

Wrong, China is, and we did a bang up job pushing Russia into Chinese arms as allies, when they never needed to be.

PS - nothing I've stated isn't verifiable online, outside of US mainstream media which is govt propaganda too, well, liberal propaganda. Also, two things can be true, I've no idea what Putin is telling Russian people, I just know that he can't afford to look weak, ego too big, and if that IS what he tells them, he ain't wrong.

**Reply by vladmandic** [reply]
guys, do we really need to go into geopolitics? i mean, its ok if you really want to, but this is not the best place for it ;)

**Reply by myndxero** [reply]
Agree, just sleep deprived and scatterbrained.

**Reply by kybercore** [reply]
> You should probably do some research, current Russia is of the US's own making. When the USSR was desolved they were more than ready to join Westerners in capitalism. But Bush then Clinton and Obama all stone-walled them from joining Nato, despite the fact that it was the offering if they reduced their nuclear arsenal for which they complied.
> 
> Putin sucks and is an asshole, and his obsession with super long tables is weird, but it was a monster of our own making, just like weak US leadership was the opportunity Putin needed to invade Crimea, which btw, has ALWAYS belonged to Russia since Russia was a thing, until we took it away. Would be similar to telling the Jews, too bad, you can't have Israel anymore.
> 
> And btw, US further exacerbated the conflict over Ukraine suggested to join Nato, that has no right to be in Nato. They're not a democracy, one of the most corrupt countries on the planet where we send our money to get laundered.
> 
> So yeah, I don't know much about Kandinsky, I don't care, it's free and open source, so not only are you an idiot for suggesting using it SUPPORTS a 'terrorist state,' you know absolutely nothing about the politics behind it. I may have missed some things, but lightyears ahead of your intelligent "magic formula help you against armed soldiers who are raping" cuz we already have that mindset infecting our population.
> 
> .fin

You have no clue. Russia is very deeply imperialist, there was zero reflection about the fact that Russia and the Nazis invaded Poland together at the start of WW2. That's just not present in Russian society. They immidiately drove up the East German military to look like the Russian one with massive shows of force after WW2 so long before Bush and Obama.

And since you forgot, Russia still occupies 20 % of Georgia, since 2008. They don't care about what the country they annex looks like, there is a reason for why Russia has expanded from approx. Caucasus all the way to Japan.

And yes, the US has done a bunch of wrong things too and has a ton of hyper-nationalist idiots but it's a joke next to the Russian society.

Russia isn't just a terrorist state, it's megalomania in the form of a country, rotten to the core. It's what happens when you have zero self awareness in politics and society about anything.

**Reply by NovaSabre** [reply]
> Kandinsky model is a product of the Russian goverment bank team. Supporting them is a supporting terrorist state

Let me guess, you live in a mud hut and you sent that with smoke signals. If not you are a hypocrite. Everything you own comes from terrorist states. rare earth metals from slaves in Africa/India/China, Odds are 90% of the things surrounding you has something made in china in them if not completely made in China.

**Reply by vladmandic** [reply]
@parsec501 sure, yes, but that topic is dead, no need for reignite it.

**Reply by NovaSabre** [reply]
> Well, you're quoting the russian state propaganda word by word: bad US, NATO expansion, blahblah. Nothing to discuss here. Russia is the civilized world enemy, and it's the main point here.

So at the end of this year when the new nations join and 70% of the world population is allied in BRICS. Will you stop doing business with those "uncivilized" countries ? You are acting like a child...

**Reply by kybercore** [reply]
> @parsec501 sure, yes, but that topic is dead, no need for reignite it.

You are right, I'll leave it at this. I just hate to see US right wingers talk over people from ex soviet countries about how they not wanting to be invaded by the same government again is a form of aggression

**Reply by ke1ne** [reply]
> > Well, you're quoting the russian state propaganda word by word: bad US, NATO expansion, blahblah. Nothing to discuss here. Russia is the civilized world enemy, and it's the main point here.
> 
> So at the end of this year when the new nations join and 70% of the world population is allied in BRICS. Will you stop doing business with those "uncivilized" countries ? You are acting like a child...

Yeah, quite civilized and nice guys https://news.yahoo.com/1-killed-8-wounded-russian-145349324.html

**Reply by NovaSabre** [reply]
The us killed just under a million people in their oil crusades... 

https://watson.brown.edu/costsofwar/figures/2021/WarDeathToll

https://en.wikipedia.org/wiki/Haska_Meyna_wedding_party_airstrike
47 civilians, including the bride, had been killed

https://en.wikipedia.org/wiki/Wech_Baghtu_wedding_party_airstrike
37 [Afghan] civilians, mostly women and children, and injuring about 27 others


I don't see you bitching about supporting the US terror state

**Reply by vladmandic** [reply]
why is this continuing? i'd hate to be the one to lock the thread, but it seems necessary.
